pkg/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -281,8 +281,8 @@ var _ = Describe("Tuned", func() {
 					netDeviceName := "!ens5"
 					netDeviceVendorID := "0x1af4"
 					netDeviceModelID := "0x1000"
-					//regex field should be: devices_udev_regex=r'^ID_MODEL_ID=0x1000[\\s\\S]*^ID_VENDOR_ID=0x1af4[\\s\\S]*^INTERFACE_NAME=(?!ens5)'
-					devicesUdevRegex := `r'\^ID_MODEL_ID=` + netDeviceModelID + `\[\\\\s\\\\S]\*\^ID_VENDOR_ID=` + netDeviceVendorID + `\[\\\\s\\\\S]\*\^INTERFACE_NAME=\(\?!` + netDeviceName + `\)'`
+					//regex field should be: devices_udev_regex=r'^ID_MODEL_ID=0x1000[\\s\\S]*^ID_VENDOR_ID=0x1af4[\\s\\S]*^INTERFACE=(?!ens5)'
+					devicesUdevRegex := `r'\^ID_MODEL_ID=` + netDeviceModelID + `\[\\\\s\\\\S]\*\^ID_VENDOR_ID=` + netDeviceVendorID + `\[\\\\s\\\\S]\*\^INTERFACE=\(\?!` + netDeviceName + `\)'`
```

Comment:
Is there any documentation for us to see the correct terms? I am referring to the /sINTERFACE_NAME/INTERFACE .

Reply:
these are device attributes (net device in our case)  in a udev format.
for example:
```
$ udevadm info /sys/class/net/enp3s0
P: /devices/pci0000:00/0000:00:1c.5/0000:03:00.0/net/enp3s0
E: DEVPATH=/devices/pci0000:00/0000:00:1c.5/0000:03:00.0/net/enp3s0
E: ID_MODEL_ID=0x8168
E: ID_VENDOR_ID=0x10ec
E: INTERFACE=enp3s0
```


tuned uses https://pyudev.readthedocs.io/en/latest/guide.html lib get these values.

Reply:
ack